tests/builder/builder_test.py

```diff
@@ -2261,8 +2261,8 @@ def test_load_kerning_bracket(ufo_module):
 
     ds = glyphsLib.to_designspace(font, minimize_glyphs_diffs=True)
     bracketed_groups = {
-        "public.kern2.foo": ["a", "a.BRACKET.wght_300_1000"],
-        "public.kern1.foo": ["x", "x.BRACKET.wght_300_1000", "x.BRACKET.wght_600_1000"],
+        "public.kern2.foo": ["a", "a.BRACKET.varAlt01"],
```

Comment:
do we need to keep both "BRACKET" and "varAlt" in the same name? Aren't the two labels supposed to convey the same meaning?
how about either just "BRACKET.01" or "varAlt01"?

Reply:
We do lose a lot of the information that was previously baked in the (arguably quite long) glyph name before.

I wonder what order does this varAlt01, 02, 03, naming etc. follows? Is it deterministic at least?

Reply:
We don't *need* to keep both, but I went this way because I think we want to keep something which is unmistakably bracket-related, for the various regex-based tests to work; and I went with `varAlt01` because that's what Glyphs adds.

Reply:
if varAlt01 is what Glyphs.app uses, I'd say let's go for that one only, and fix related tests.

In what order are the digit suffix in varAlt01, 02, 03 assigned and incremented, in what order? Is it "don't ask too complicated doesn't actually matter" kind of order?

Reply:
i see, it depends on stably ordered stuff like glyph order, layer order and order of components within each layer, so at least it's deterministic, it just as eloquent as the previous naming scheme